src/storage/txn/actions/prewrite.rs

```diff
@@ -23,9 +26,10 @@ pub fn prewrite<S: Snapshot>(
     mutation: Mutation,
     secondary_keys: &Option<Vec<Vec<u8>>>,
     is_pessimistic_lock: bool,
-) -> Result<TimeStamp> {
+    need_old_value: bool,
```

Comment:
I think this can be added to `TransactionProperties` since it is a property of the whole transaction, not individual mutations.